lesson-11/main.js

```diff
@@ -1,43 +1,96 @@
-const peopleInput = document.querySelector('.header__people input');
+const peopleInput = document.querySelector('.header__people');
+const adultsInput = document.querySelector('.header__input_adults');
+const childrenInput = document.querySelector('.header__input_children');
+const roomsInput = document.querySelector('.header__input_rooms');
 const peopleFilter = document.querySelector('.header__people_filter');
 
-const focusPeople = (e) => {
-    peopleFilter.style.display = 'block';
-    peopleInput.setAttribute('placeholder', '0 Adults — 0 Children — 0 Room')
-}
-peopleInput.addEventListener('focus', focusPeople);
+const focusPeople = () => {
+  peopleFilter.style.display = 'block';
+  adultsInput.setAttribute('placeholder', '0 Adults     — ');
+  childrenInput.setAttribute('placeholder', '0 Children   — ');
+  roomsInput.setAttribute('placeholder', '0 Room');
```

Comment:
This code is unnecessary. We should change initial input value

Reply:
Deleted this code.